bin/install.sh

```diff
@@ -161,6 +176,7 @@ install_init_vars() {
     : ${install_server:=github}
     : ${install_tmp_dir:=/var/tmp}
     : ${install_verbose:=}
+    : ${install_proprietary_key:=missing-proprietary-key}
```

Comment:
Oof, Google doesn't like my searches for what this means. Thankfully [this](https://www.tldp.org/LDP/abs/html/special-chars.html) had the answer near the top of the page when just searching for ":".